Kommunicate/Classes/KMConversationViewController.swift

```diff
@@ -442,6 +442,7 @@ extension KMConversationViewController {
                     self?.showRatingView()
                     return
                 }
+                self?.showRatingView()
                 self?.show(feedback: previousFeedback)
```

Comment:
As we are showing the rating view, we don't have to show previous feedback for now, so you can remove
`show(feedback: previousFeedback)` line.